From 36ebee8349cd52debf0682c7cf21853c433ca4aa Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 23:11:09 -0400
Subject: [ethernet] cxgb4vf: Adds device Id for few more Chelsio adapters

Message-id: <1410477072-95160-5-git-send-email-svemuri@redhat.com>
Patchwork-id: 93899
O-Subject: [RHEL7.1 PATCH BZ 1070945 4/7] cxgb4vf: Adds device Id for few more Chelsio adapters
Bugzilla: 1070945
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 63ea7dce8ce6de8d9d6b36b835219b38655b13e8)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
index 1d0fe9b..5285928 100644
--- a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
@@ -2944,6 +2944,14 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4vf_pci_tbl) = {
  CH_DEVICE(0x5811, 0),   /* T520-lp-cr */
  CH_DEVICE(0x5812, 0),   /* T560-cr */
  CH_DEVICE(0x5813, 0),   /* T580-cr */
+ CH_DEVICE(0x5814, 0),   /* T580-so-cr */
+ CH_DEVICE(0x5815, 0),   /* T502-bt */
+ CH_DEVICE(0x5880, 0),
+ CH_DEVICE(0x5881, 0),
+ CH_DEVICE(0x5882, 0),
+ CH_DEVICE(0x5883, 0),
+ CH_DEVICE(0x5884, 0),
+ CH_DEVICE(0x5885, 0),
  { 0, }
 };
 
-- 
1.7.1